tests/python/test_bitmasked.py

```diff
@@ -256,3 +256,32 @@ def func():
 
     func()
     assert s[None] == 7
+
+
+@ti.test(require=ti.extension.sparse)
+def test_bitmasked_2d_power_of_two():
+    some_val = ti.field(dtype=float)
+    width, height = 10, 10
+    total = width * height
+    ptr = ti.root.bitmasked(ti.ij, (width, height))
+    ptr.place(some_val)
+    num_active = ti.field(dtype=int, shape=())
+
+    @ti.kernel
+    def init():
+        num_active[None] = 0
+        for x, y in ti.ndrange(width, height):
+            ti.activate(ptr, [x, y])
```

Comment:
Is it necessary to manually call `activate()`?

Reply:
```suggestion
```